pkg/runtime/grpc/grpc-runtime.go

```diff
@@ -110,35 +138,50 @@ func (r *Runtime) Close() error {
 }
 
 func (r *Runtime) ParamDescs() params.ParamDescs {
```

Comment:
I think these should be `GlobalParamDescs`

Reply:
I had that in GlobalParamDescs first, but if you run gadgetctl as a (web-)server (to be added later on™), it might make sense to choose your target after the server has been initialized.